cobalt/doc/cvals.md

```diff
@@ -457,3 +457,42 @@ query):
     writes.
 *   **Starboard.StorageWriteRecord.BytesWritten** - Bytes written to storage
     record.
+
+### CPU
+
+#### PublicCVals
+
+*  **CPU.Total.Usage.IntervalSeconds.[INTERVAL_SECONDS]** - CPU usage in
```

Comment:
Is `INTERVAL_SECONDS` arbitrary ? e.g. I can call it with `cval.get('CPU.Total.Usage.IntervalSeconds.10')` and `cval.get('CPU.Total.Usage.IntervalSeconds.60') if i want to, and also interleave the calls ? It's not super clear from reading the doc here.

It would also be helpful to have a demo page under `cobalt/demos/content` that shows how to use it.

Reply:
Nit #1: The JS-inclined folks will read `INTERVAL` as in `setInterval()`. My mind immediately goes to heartbeat-like looping behavior.

Reply:
Nit #2: Shouldn't `IntervalSeconds` be `IntervalMicroseconds` here?